styles/main.css

```
@@ -2,3 +2,97 @@
 :root {
   --bs-font-sans-serif: 'Roboto', system-ui, -apple-system, "Segoe UI", "Helvetica Neue", Arial, "Noto Sans", "Liberation Sans", sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol", "Noto Color Emoji";
 }
+
+
+div {
+  display: block;
+}
+
+/********************/
+/*    Customers     */
+/********************/
+
+.blue {
```

Would be better if you set the name like this:

```suggestion
.text-blue {
```

---

Reviewed and corrected